src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/NullPathConverter.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+using System.Collections.Generic;
+using ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+using System.Collections.ObjectModel;
+
+internal class NullPathConverter : IPathConverter
```

Comment:
[optional] Maybe there is no need to allow multiple instance of this type as it's just a pass-through type.

Reply:
Used static instance.